XamarinCommunityToolkitSample.iOS/Xamarin.CommunityToolkit.Sample.iOS.csproj

```diff
@@ -25,7 +25,7 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <MtouchArch>x86_64</MtouchArch>
-    <MtouchLink>None</MtouchLink>
+    <MtouchLink>SdkOnly</MtouchLink>
```

Comment:
Do we want to set Linker to SdkOnly by default for Debug?

Reply:
I think it's a good idea. That way we can reflect on how the lib will behave in release builds and the person that collaborates with will see any issue sooner.